config/locales/en.yml

```
@@ -139,6 +139,10 @@ en:
     password: Password
     change: Change
     mfa: Multi-factor authentication
+    software_token: Multi-Factor Authentication Present
```

Don't capitalize - https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#capitalisation
```suggestion
    software_token: Multi-factor authentication present
```